integrations/Pendo/browser.js

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-underscore-dangle */
+import logger from "../../utils/logUtil";
+import ScriptLoader from "../ScriptLoader";
+import { flattenJsonPayload } from "../../utils/utils";
+
+class Pendo {
+  constructor(config, analytics) {
+    this.analytics = analytics;
+    this.apiKey = !config.apiKey ? "" : config.apiKey;
+    this.isDebugMode = config.enableDebugMode;
+    this.name = "PENDO";
+    this.setIntervalHandler = undefined;
+    logger.debug("Config ", config);
+  }
+
+  init() {
+    window.pendo = {};
+    window.pendoCli = undefined;
+    window.pendo.apiKey = this.apiKey;
+    ScriptLoader(
```

Comment:
why not use the initialisation script from here: https://developers.pendo.io/docs/?bash#introduction ?
also their script seems similar to us where the 'initialize','identify','updateOptions','pageLoad' are already present even when the script is not ready. (Also  include track there in the list as I think it got supported in later version). Then our isReady()  and isLoaded() can always return true

Reply:
Initialized scripts in init function.